buddypress/members/single/edit.php

```diff
@@ -604,8 +617,19 @@
             </label>
         </section>
         -->
```

Comment:
Let's remove this commented code since you moved the form into it's own template